packages/vaadin-grid/test/resizing.test.js

```diff
@@ -167,10 +167,10 @@ describe('height by rows', () => {
       </vaadin-grid>
     `);
     const rows = 100;
-    grid.items = grid.items = Array(...new Array(rows)).map(() => {});
+    grid.items = grid.items = Array.from({ length: rows });
```

Comment:
Am I missing something here, or is the double assignment not necessary?
```suggestion
    grid.items = Array.from({ length: rows });
```

Reply:
yeah, forgot remove that, seems unnecessary